### DISCUSSION TITLE
Is there a way to force default locale in URL

### DISCUSSION THREAD

**Discussion by lipoolock** [vercel/next.js#20845] [Help]
Hi
I found many subjects, requests related to that behaviour 
(particularly this one https://github.com/vercel/next.js/discussions/18419) but never found an official answer from NextJs team

The simple pattern of this request is to have a way to never fallback on `/` but always on `/[defaultLocale]`

I have a classic example here https://codesandbox.io/s/misty-water-ud4rt

I would like to have [en] navigation forced with `/en/[my-path]` instead of `/[my-path]`

Any insight to help us to go forward with that topic ?

Thanks ;)

**Comment by AssisrMatheus** [comment]
I think this is related to https://github.com/vercel/next.js/discussions/18419
